performance-tests/pom.xml

```diff
@@ -64,6 +64,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
No longer comes transitively via the **testutils** module so had to be explicitly added.